src/main/java/hudson/plugins/gradle/injection/GitScmListener.java

```diff
@@ -68,8 +68,8 @@ public void onCheckout(
     private static boolean isVCSInjectionFilteringNotApplied(SCM scm, InjectionConfig config) {
         return config.isDisabled()
                 || InjectionUtil.isInvalid(InjectionConfig.checkRequiredUrl(config.getServer()))
-                || config.getParsedInjectionVcsRepositoryPatterns() == null
-                || config.getParsedInjectionVcsRepositoryPatterns().isEmpty()
+                || config.getParsedVcsRepositoryFilter() == null
+                || config.getParsedVcsRepositoryFilter().isEmpty()
                 || vcsRepositoryUrlMatches(config, scm);
```

Comment:
I find method naming confusing here. `isVCSInjectionFilteringNotApplied` means no VSC filtering, but the last check does the matching (at least according to its name). Also now we can have positive and negative matching.

Reply:
What about calling it `shouldInjectionContinue` and also rename `vcsRepositoryUrlMatches` to `vcsRepositoryUrlInjectionEnabled`?

Reply:
Yep, I was also thinking about splitting it.

Reply:
Done